build/Dockerfile

```
@@ -42,4 +42,6 @@ RUN curl -L https://ftp.openbsd.org/pub/OpenBSD/LibreSSL/libressl-2.8.2.tar.gz >
     rm -rf /tmp/libressl-2.8.2 /tmp/libressl.tar.gz
 
 ENV JAVA_HOME=/usr/lib/jvm/java-1.8.0
+ENV CC=/opt/rh/devtoolset-8/root/usr/bin/gcc
```

```suggestion
ENV CC=gcc
```
I don't think passing the full path to gcc is a good idea here. IIRC calling into this gcc will fail if `scl enable` wasn't called correctly. So this will make upgrading the devtoolset a bit more likely to introduce issues (as now we need to change two more lines). Is there a particular reason you want to pass the full path?

---

@mpilman distcc in a secure mode (which we must enable internally) is required to use the full path. Also, as the build docker for our software, I think it's fair to be specific (using absolute path) to define the compiler we expect to use to build the software.

---

I see. The main thing I am a bit worried about is that upgrading devtoolset will now require more changes. But I guess this isn't a biggy. Also I am not familiar with distcc so I wasn't aware of the first point.